modules/openshift-cluster-maximums-major-releases.adoc

```diff
@@ -55,25 +55,26 @@ Red Hat does not provide direct guidance on sizing your {product-title} cluster.
 | 2,000
 
 | Number of build configs
 | 12,000
 
 | Number of custom resource definitions (CRD)
-| 512 ^[7]^
+| 512 ^[8]^
 
 |===
 [.small]
 --
 1. Pause pods were deployed to stress the control plane components of {product-title} at 2000 node scale. The ability to scale to similar numbers will vary depending upon specific deployment and workload parameters.
 2. The pod count displayed here is the number of test pods. The actual number of pods depends on the application's memory, CPU, and storage requirements.
-3. This was tested on a cluster with 100 worker nodes with 500 pods per worker node. The default `maxPods` is still 250. To get to 500 `maxPods`, the cluster must be created with a `maxPods` set to `500` using a custom kubelet config. If you need 500 user pods, you need a `hostPrefix` of `22` because there are 10-15 system pods already running on the node. The maximum number of pods with attached persistent volume claims (PVC) depends on storage backend from where PVC are allocated. In our tests, only {rh-storage} v4 (OCS v4) was able to satisfy the number of pods per node discussed in this document.
-4. When there are a large number of active projects, etcd might suffer from poor performance if the keyspace grows excessively large and exceeds the space quota. Periodic maintenance of etcd, including defragmentation, is highly recommended to free etcd storage.
-5. There are a number of control loops in the system that must iterate over all objects in a given namespace as a reaction to some changes in state. Having a large number of objects of a given type in a single namespace can make those loops expensive and slow down processing given state changes. The limit assumes that the system has enough CPU, memory, and disk to satisfy the application requirements.
-6. Each service port and each service back-end has a corresponding entry in iptables. The number of back-ends of a given service impact the size of the endpoints objects, which impacts the size of data that is being sent all over the system.
-7. {product-title} has a limit of 512 total custom resource definitions (CRD), including those installed by {product-title}, products integrating with {product-title} and user created CRDs. If there are more than 512 CRDs created, then there is a possibility that `oc` commands requests may be throttled.
+3. This was tested on a cluster with 31 servers: 3 control planes, 2 infrastructure nodes, and 26 worker nodes. The default `maxPods` is still 250. To get to 2,500 pods per node, the cluster must be created with `maxPods` set to `2500` using a custom kubelet config. If you need 2,500 user pods, you need a `hostPrefix` of `20` because there are 10-15 system pods already running on the node. The maximum number of pods with attached persistent volume claims (PVC) depends on the storage backend from where PVC are allocated. In our tests, only {rh-storage} v4 (OCS v4) was able to satisfy 2,500 of pods per node.
```

Comment:
Need some changes here.

"The maximum number of pods with attached persistent volume claims (PVC) depends on the storage backend from where PVC are allocated.  In our tests, only {rh-storage} v4 (OCS v4) was able to satisfy 2,500 of pods per node." Can be removed as it is irrelevant since I did not use OCS nor pods with PVCs in my tests.

I would swap the order of "The default maxPods..." and "... hostPrefix ..." sentences because hostPrefix is an install-time configuration and maxPods is a Day 2 post-install operation.

Are we planning to link directly to the blog from the docs?

https://cloud.redhat.com/blog/running-2500-pods-per-node-on-ocp-4.13